style.css

```diff
@@ -37,16 +49,37 @@ body {
   width: 100%;
   height: 100%;
 }
+.c-burger-menu__inner {
+  position: absolute;
```

Comment:
Try grouping the related items to make this easier to reason about, i.e. all flex items together